src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -56,21 +58,17 @@ public ResponseEntity fetchBlogs(@PathVariable("name") String name) {
      */
     @PostMapping(value = "/disorders/{name}/blog", consumes = "application/json", produces = "application/json")
     public Blog createBlog(@PathVariable("name") String name, @RequestBody Blog blog) {
-        // we're going to need a user id passed here as well. for now, let's not worry about it
-
-        // for not, just return created blog as a stub
         return blog;
     }
 
     /**
      * Delete a blog
-     * @param name
-     * @param blog
+     * @param //name
+     * @param //blog
```

Comment:
errors were being thrown by these params listed in JavaDoc. I just commented them out

Reply:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.